### DISCUSSION TITLE
create dynamic number of MovablePoints

### DISCUSSION THREAD

**Discussion by trong-tran** [stevenpetryk/mafs#107] [Help] [answered]
I would like to create n points and connect them with lines. (Like an n-polygon). The problem is useMovablePoint is a hook and can not be used in a callback. Please give an example in which movablePoints are created dynamically.
Thanks a lot

**Comment by stevenpetryk** [comment, marked as answer]
Hey there, if you scroll to the bottom of the [docs page for movable points](https://mafs.dev/guides/interaction/movable-points) it covers that exact use case with an example close to what you're after.

**Reply by trong-tran** [reply]
Thanks, i see it. Thanks a lot.
